src/apis/btc-core.ts

```diff
@@ -52,19 +52,20 @@ export class DefaultBTCCoreAPI implements BTCCoreAPI {
     constructor(network: string = "mainnet") {
         let basePath = "";
         switch (network) {
-        case "mainnet":
-            basePath = mainnetApiBasePath;
-            break;
-        case "testnet":
-            basePath = testnetApiBasePath;
-            break;
-        default:
-            basePath = network;
+            case "mainnet":
```

Comment:
Seems like the linter on the pipeline and your local one do not agree here. Can you fix this before merging, please?